cms/models.py

```diff
@@ -70,3 +73,27 @@ def get_absolute_url(self):
 
     def __str__(self):
         return self.title
+
+
+class BlogPage(BasePage):
+    author = models.ForeignKey(
+        User, on_delete=models.CASCADE, verbose_name="Erstellt von"
+    )
+    components = models.ManyToManyField(Component, blank=True, verbose_name="Lösungen")
+    title_image = models.ImageField(
+        upload_to="blog/", blank=True, verbose_name="Titelbild"
+    )
+    created = models.DateTimeField("Erstellt", auto_now_add=True)
+    slug = models.SlugField(default="", unique=True, max_length=200, verbose_name="Url")
+
+    # Maybe some approve mechanism?
```

Comment:
a `published` bool would be good

Reply:
is already there since its parent is the base page, which has this field.